cli/cmd/plugin/ui/web/tanzu-ui/src/App.tsx

```diff
@@ -6,7 +6,7 @@ import { Routes, Route } from 'react-router-dom';
 import { APP_ENV_CHANGE, AppActionNames } from './state-management/actions/App.actions';
 import { NavRoutes } from './shared/constants/NavRoutes.constants';
 import { Store } from './state-management/stores/Store';
-import AwsManagementClusterSimple from './views/management-cluster/AwsManagementClusterSimple';
```

Comment:
I thought we were using `Simple` here to distinguish from our anticipated `Advanced` flow in the future. (Maybe `Basic` is better?)